libs/model/src/community/GetCommunities.query.ts

```diff
@@ -0,0 +1,219 @@
+import { type Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { z } from 'zod';
+import { models } from '../database';
+
+export function GetCommunities(): Query<typeof schemas.GetCommunities> {
+  return {
+    ...schemas.GetCommunities,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const {
+        base,
+        include_node_info,
+        stake_enabled,
+        has_groups,
+        cursor,
+        limit,
+        order_by,
+        order_direction,
+      } = payload;
+      // NOTE 8/7/24: as a result of having to provide a preprocessed string instead of
+      //  being able to provide an array, we must explicitly set tag_ids as type number[]
+      //  here. The input parser will handle incorrectly formatted strings (see the
+      //  tags.spec.ts file where it is exercised) so this is safe to do.
+      const tag_ids = payload.tag_ids as number[];
```

Comment:
looks like there is no need to cast here... I see optional number[] being inferred

Reply:
Resolved